SSDReco/MakeSSDClusters_module.cc

```diff
@@ -57,16 +57,14 @@ class emph::MakeSSDClusters : public art::EDProducer {
   int run,subrun,event;
   std::vector<int> station, sens, view, ndigits, width, timerange;
   std::vector<float> avgadc, avgstrip, wgtavgstrip, wgtrmsstrip;
-  static const int NPlanes = 20;
+  
+  int NPlanes;
   static const int NStations = 8;
-  static const int MaxSensPerSta = 6;
-  int ncluster[NPlanes];
-
-  std::vector<float> clustdist[NPlanes];
+  static const int MaxPlnsPerSta = 3;
+  static const int MaxSensPerPln = 2;
+  std::vector<int> ncluster;
+  //  std::vector<float> clustdist;
```

Comment:
This was something Robert added, do we know that it doesn't have a use-case anymore? If so, the stuff that has been commented out related to it should be removed.

Reply:
Yes, I've confirmed with Robert.  Removed.